jdaviz/core/template_mixin.py

```diff
@@ -906,6 +912,16 @@ def remove_filter(self, *filters):
         self.filters = [f for f in self.filters
                         if (f not in filters and getattr(f, '__name__', '') not in filters)]
 
+    @observe('filters')
+    def _update_items(self, msg={}):
+        all_items = [self._to_item(opt) for opt in self.manual_options]
+        self.items = [item for item in all_items if self._is_valid_item(item)]
```

Comment:
adding this to the base class then requires renaming all the inheriting classes to have their methods renamed so that they properly override this instead of resulting in duplicated functionality.